src/govuk/components/character-count/character-count.mjs

```diff
@@ -30,7 +31,21 @@ function CharacterCount ($module, config) {
   }
 
   var defaultConfig = {
-    threshold: 0
+    threshold: 0,
+    i18n: {
+      // Characters
+      charactersUnderLimitOne: 'You have %{count} character remaining',
+      charactersUnderLimitOther: 'You have %{count} characters remaining',
+      charactersAtLimit: 'You have no characters remaining',
+      charactersOverLimitOne: 'You have %{count} character too many',
+      charactersOverLimitOther: 'You have %{count} characters too many',
+      // Words
+      wordsUnderLimitOne: 'You have %{count} word remaining',
+      wordsUnderLimitOther: 'You have %{count} words remaining',
+      wordsAtLimit: 'You have no words remaining',
+      wordsOverLimitOne: 'You have %{count} word too many',
+      wordsOverLimitOther: 'You have %{count} words too many'
+    }
```

Comment:
All of these now need to be in the JSDoc! 😱

Reply:
Added, but I may want to dig a bit further into it as it makes massive lines 😬 @colinrotherham would you have any suggestions on how to make it a bit more manageable?